builtins/amp-img.js

```diff
@@ -145,7 +148,9 @@ export class AmpImg extends BaseElement {
 
     this.propagateAttributes(ATTRIBUTES_TO_PROPAGATE, this.img_);
     guaranteeSrcForSrcsetUnsupportedBrowsers(this.img_);
-    if (isExperimentOn(this.getAmpDoc().win, 'amp-img-auto-sizes')) {
+    this.autoGenerateSizes_ = isExperimentOn(this.getAmpDoc().win,
```

Comment:
This can be read in the constructor, use `this.win`

Reply:
Ah, got it. Thanks!

Reply:
Okie. This is done, and the tests are updated accordingly.